src/services/notifications.js

```diff
@@ -0,0 +1,40 @@
+import _ from 'lodash';
+import moment from 'moment';
+import sessions from './sessions';
+
+const fallbackEmail = 'admin@example.org'; // TODO where to send these really?
+
+function notifyUnreviewed() {
+  sessions.list({ 'sessions.reviewed': false }, 2000, 0, 'desc')
```

Comment:
Actually, on further thoughts we probably should just send anyways, even for fresh sessions. If we filter, we'd need to change the text to be "you have X overdue sessions", and we can't with good consciousness say something created on Sunday is overdue on Monday morning before the workday.